src/angular/controllers/__snapshots__/ctrl-widget-catalog.test.js.snap

```diff
@@ -0,0 +1,188 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Since the Angular version of the widget catalog has been deprecated in favor of React, this file isn't being used any more. I'll add another commit to remove it.